sql/core/src/test/resources/sql-tests/results/explain-aqe.sql.out

```diff
@@ -925,10 +925,7 @@ Created By [not included in comparison]
 Type: MANAGED
 Provider: PARQUET
 Location [not included in comparison]/{warehouse_dir}/explain_temp1
-Schema: root
--- key: integer (nullable = true)
--- val: integer (nullable = true)
-), false
+Schema: struct<key:int,val:int>), false
```

Comment:
My last concern is the loss of the nullable info. How does other databases do?

Reply:
It seems other databases do not include `Schema` when `SHOW EXTENDED TABLES`:
![image](https://user-images.githubusercontent.com/5399861/120504345-a30f6b80-c3f6-11eb-9bd9-418c9846a4b1.png)
![image](https://user-images.githubusercontent.com/5399861/120508017-ecad8580-c3f9-11eb-87de-afe01c8bd193.png)

Reply:
hive> SHOW TABLE EXTENDED LIKE '*';
OK
tableName:spark_32976
owner:yumwang
location:file:/tmp/spark/spark_32976
inputformat:org.apache.hadoop.mapred.TextInputFormat
outputformat:org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat
columns:struct columns { i32 id, string name}
partitioned:true
partitionColumns:struct partition_columns { string part}
totalNumberFiles:unknown
totalFileSize:unknown
maxFileSize:unknown
minFileSize:unknown
lastAccessTime:unknown
lastUpdateTime:unknown